docs/reference/commandline/stack_deploy.md

```diff
@@ -57,6 +57,8 @@ Creating service vossibility_ghollector
 Creating service vossibility_lookupd
 ```
 
+The Compose file can also be provided as standard input with `--compose-file -`.
```

Comment:
I agree, then it doesn't need to be specifically dealt with in the reference file at all, unless you wanted to add an example (which would probably be good to have).